Proxy/proxy.py

```diff
@@ -262,7 +262,7 @@ def proxy():
 
             if limit is None:
                 # Optional
-                limit = len(queDict[port])
+                limit = len(getDicts[station][port])
```

Comment:
@reillyeon is this you adding this or the automated tests? This was done in https://github.com/reillyeon/Faraday-Software/commit/6060044dedfafc2afb574df8bf3263e7e955c0bf where you fixed undefined variables. Is this considered one?